js/tji-datasets-explore.js

```diff
@@ -474,18 +474,30 @@ TJIChartView.prototype.create_filters = function() {
   this.state.active_filters = this.ui.$form.serializeArray();    
 }
 
+TJIChartView.prototype.create_filter_legend = function(filter_name) {
+  return jQuery('<legend />', {
+      class: 'js-filter-legend tji-chartview-filters__filter-legend'
+    })
+    .text(filter_name.replace(/_/g, " ").toLowerCase())
+    .append('<i class="fas fa-caret-down"></i>');
+}
+
 TJIChartView.prototype.create_filter_checkboxes = function(filter) {
   var that = this;
-  var fieldset = jQuery('<fieldset><legend>' + filter.name.replace(/_/g, " ") + '<i class="fas fa-caret-down"></i></legend></fieldset>');
+  var fieldset = jQuery('<fieldset/>');
+  var legend = this.create_filter_legend(filter.name);
+  var filterset = jQuery('<div class="js-filter-set tji-chartview-filters__filter-set"><a class="js-toggle-select">Select All</a> / <a class="js-toggle-unselect">Unselect All</a></div>');
```

Comment:
filterset wraps the portion of the filters that needs to be collapsed or expanded.